package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Not sure where this one came from, but we shouldn't need this in the root folder. We've already got `src/dymaptic.GeoBlazor.Core/package.json`